script/bootstrap-st2

```diff
@@ -14,6 +14,7 @@ show_help() {
   echo "Usage: $0 [OPTION]..."
   echo
   echo "  -a,  --answers     Supply installer with an answer file"
+  echo "  -r,  -- ref        Supply a SHA or TAG to use as installer version"
```

Comment:
I left it out intentionally. It appears currently this feature is not a customer facing one. So let's not document it. Customers should only use -t option.